app/src/pages/contact.tsx

```diff
@@ -0,0 +1,100 @@
+import type { GetServerSideProps, NextPage } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { ChangeEvent, useState } from 'react'
+import NumberFormat from 'react-number-format'
+import styled from 'styled-components'
+
+import ButtonLink from '@components/ButtonLink'
+import TextInput from '@components/TextInput'
+
+const Contact: NextPage = () => {
+  const { t } = useTranslation('common')
+  const [form, setForm] = useState({
+    firstName: '',
+    lastName: '',
+    phone: '',
+    other: '',
+  })
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { value, id }: { value: string; id: string } = e.target
+    const castId = id as keyof typeof form
+
+    setForm({
+      ...form,
+      [castId]: value,
+    })
+  }
+
+  return (
+    <form>
+      <TextInput
+        handleChange={handleChange}
+        id="firstName"
+        label={t('Contact.firstName')}
+        value={form.firstName}
+      />
+      <br />
+      <TextInput
+        handleChange={handleChange}
+        id="lastName"
+        label={t('Contact.lastName')}
+        value={form.lastName}
+      />
+      <br />
+      <Label className="usa-label" htmlFor="phone">
+        {t('Contact.phone')}
+      </Label>
+      <PhoneInput
```

Comment:
Out of curiosity, where does this get imported into the file? 👀 How come this doesn't throw an error?

Reply:
PhoneInput and Helper are styled-components defined at the bottom of this file :)